src/BuildScriptGeneratorCli/Commands/BuildCommand.cs

```diff
@@ -270,15 +338,15 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
                 }
 
                 // Not using IConsole.WriteErrorLine intentionally, to keep the child's error stream intact
-                console.Error.WriteLine(line);
+                console.WriteLine(line);
```

Comment:
I'm wondering if moving from console.Error.WriteLine to console.WriteLine can have some side effects, as we won't be sending data to stderr anymore. Is there any equivalent we could use? If yes, why not using it?

Reply:
There is a `console.Error.Write`. I think this was some early changes that i kept the old method. I'll update them in the next commit.

Reply:
Thanks!